module/behaviour/strategy/KeyboardWalk/src/KeyboardWalk.hpp

```diff
@@ -73,7 +73,7 @@ namespace module::behaviour::strategy {
         void look_up();
         void look_down();
         void walk_toggle();
-        void quit();
+        static void quit();
```

Comment:
This seems wrong?

Reply:
Although this function doesn't seem to be called at all

Reply:
I have screwed up this branch. These changes which you have reviewed are all basically from another PR which I accidentally merged into this one, without merging master into the base branch.

Reply:
#439 is where these comments should go once I remove these changes from this PRs diffs. Trent addressed most of these things on #433